apps/web/src/app/settings/registration/page.tsx

```diff
@@ -1,18 +1,21 @@
-import AccountSettings from "@/components/settings/AccountSettings";
-import { users } from "db/schema";
-import { eq } from "db/drizzle";
+import RegisterFormSettings from "@/components/settings/RegistrationForm/RegisterFormSettings";
 import { auth } from "@clerk/nextjs";
 import { db } from "db";
+import { eq } from "db/drizzle";
+import { users } from "db/schema";
 import { redirect } from "next/navigation";
 
 export default async function Page() {
 	const { userId } = auth();
+	if (!userId) throw new Error("User not found");
 	const user = await db.query.users.findFirst({
-		with: { registrationData: true },
+		with: {
+			registrationData: true,
+			profileData: true,
+		},
 		where: eq(users.clerkID, userId!),
```

Comment:
nit: why does `userId` need to have the `!` behind it? It should already be not null at this point.

Reply:
This was to appease TypeScript. It thinks user may be undefined in the props argument.

Reply:
I will take a look at this.